### PR TITLE
Add invariant mass selection

### PR DESCRIPTION
**Pull request by lbariogl**

### CODE REVIEW

File: PWGCF/DataModel/FemtoDerived.h [hunk 1]
```diff
@@ -115,8 +115,8 @@ DECLARE_SOA_TABLE(FemtoDreamParticles, "AOD", "FEMTODREAMPARTS",
                   femtodreamparticle::PIDCut,
                   femtodreamparticle::TempFitVar,
                   femtodreamparticle::Indices,
-                  femtodreamparticle::MinvLambda,     // TEMPORARY
-                  femtodreamparticle::MinvAntiLambda, // TEMPORARY
+                  femtodreamparticle::MinvLambda,
+                  femtodreamparticle::MinvAntiLambda,
```
Comment (victor-gonzalez) [inline PWGCF/DataModel/FemtoDerived.h:119]: The names are a bit misleading, `InvMLambda` and `InvMAntiLambda` perhaps?

File: PWGCF/FemtoDream/FemtoDreamV0Selection.h [hunk 2]
```diff
@@ -264,6 +280,13 @@ bool FemtoDreamV0Selection::isSelectedMinimal(C const& col, V const& v0, T const
   const float dcaDaughv0 = v0.dcaV0daughters();
   const float cpav0 = v0.v0cosPA(col.posX(), col.posY(), col.posZ());
 
+  const float invMassLambda = v0.mLambda();
+  const float invMassAntiLambda = v0.mAntiLambda();
+
+  if ((invMassLambda < fInvMassLowLimit or invMassLambda > fInvMassUpLimit) and (invMassAntiLambda < fInvMassLowLimit or invMassAntiLambda > fInvMassUpLimit)) {
```
Comment (victor-gonzalez) [inline PWGCF/FemtoDream/FemtoDreamV0Selection.h:286]: So now you reject when both Lambda AND ALambda are out of the limits but you accept that the Lambda is within and the ALambda is outside the limits or the other way around. Right?

**Review by victor-gonzalez — COMMENTED** [review]
Check my comments and if there is nothing to change I'll approve it

File: PWGCF/FemtoDream/FemtoDreamV0Selection.h (hunk 2 above)
Comment (lbariogl) [reply PWGCF/FemtoDream/FemtoDreamV0Selection.h:286]: Correct

File: PWGCF/DataModel/FemtoDerived.h (hunk 1 above)
Comment (lbariogl) [reply PWGCF/DataModel/FemtoDerived.h]: I adopted the same nomenclature used in the V0 tables

**Review by victor-gonzalez — APPROVED** [review]

**Review by victor-gonzalez — APPROVED** [review]

**Pull request merged**